### DISCUSSION TITLE
Daily 03/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#168] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando la plataforma, viendo si hay temas nuevos y organizandome para realizar las actividades que me faltan completar. Además empecé a ver un poco de como se haría la conexion con el backend. Esta parte me cuesta bastante.
Hoy voy a terminar con los apunte de la reunión del sábado para subirlos. Seguiré viendo lo del backend. Por la tarde voy a participar de la clase sincrónica del instituto.

**Comment by BraianTroncoso** [comment]
Ayer estuve viendo la plataforma también estuve viendo múltiples vídeos para el tema del backend y practicando un poco.
Hoy voy a participar de la clase sincrónica y sigo estudiando.

**Comment by erick-lpz** [comment]
Ayer estaba buscando como mejorar el dashboard_admin para que tenga mas funcionalidad.
Hoy busque diferentes métodos para la conexión con la base de datos y ver cuál es la mas factible de usar.

**Comment by Valentino399** [comment]
Hoy estuve haciendo un curso y practicando el código de Java para poner en practica en el código de la aplicación

**Comment by marceloedelgado** [comment]
Ayer estuve haciendo algunas pruebas en un entorno fuera del proyecto con Recycler views.

Hoy estuve capacitandome viendo videos sobre  Recycler view para crear las tarjetas de los profesionales.

Tuve problemas para listar en la primer prueba que hice

**Comment by MarielaGomez** [comment]
Estuve practicando java poo, también con un proyecto paralelo para ir volcando lo que voy investigando y viendo de tutoriales, se dificulta el poco tiempo para intentar entender todo tan rápido
